src/core/common.rs

```diff
@@ -102,26 +102,6 @@ pub mod Ids {
 const REVER_STRING: &str = "(?P<ident>[A-Z]{4})(?P<major>[0-9a-f])(?P<minor>[0-9a-f])(?P<kind>[A-Z]{4})(?P<size>[0-9a-f]{6})_";
 const IDENTS: &[&str] = &[Identage::ACDC, Identage::KERI];
 const SERIALS: &[&str] = &[Serialage::JSON];
-const ILKS: &[&str] = &[
```

Comment:
I believe I use these externally

Reply:
I may be wrong though, I may just use `Ilkage` actually

Reply:
Oh I wasn't aware that this library was in use yet.  Is it consumed in one of the public WebOfTrust tools or in a private project?

Reply:
It's consumed by my company, Qui Identity. I have a KERI/ACDC implementation in Rust in a library we make use of.

Reply:
Just checked, I do only use Ilkage:

```
❯ rg ILKS .
❯ rg Ilkage .
./src/keri/verification.rs
4:use cesride::{common::Ilkage, data::Value, Matter, Sadder, Saider, Seqner, Serder};
11:        Ilkage::vcp => serder.pre(),
12:        Ilkage::iss | Ilkage::rev => serder.ked()["ri"].to_string(),

./src/keri/parsing.rs
4:    common::{Identage, Ilkage},
47:                        Ilkage::icp | Ilkage::rot | Ilkage::ixn => {
77:                        Ilkage::vcp | Ilkage::iss | Ilkage::rev => match group {
... and so forth
```